### PR TITLE
feat: Update Hilla integration page with V24.4 unifications

### PR DESCRIPTION
**Pull request by mshabarov**

### CODE REVIEW

File: articles/flow/integrations/hilla.adoc [hunk 1]
```diff
@@ -1,265 +1,101 @@
...
-];
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.component.html.Paragraph;
+import com.vaadin.flow.component.orderedlayout.VerticalLayout;
+import com.vaadin.flow.component.textfield.TextField;
+import com.vaadin.flow.router.Route;
+
+@Route("hello")
+public class HelloView extends VerticalLayout {
+    public MainView() {
```
Comment (tltv) [inline articles/flow/integrations/hilla.adoc:90]:
```suggestion
    public HelloView() {
```

File: articles/flow/integrations/hilla.adoc [hunk 2]
```diff
@@ -1,265 +1,101 @@
...
+
 === Run the Application
 
-Run the application using `mvn spring-boot:run` and open http://localhost:8080 in your browser.
+Run the application using `mvn spring-boot:run`. Then open `http://localhost:8080` in your browser.
+
+Once you add a frontend view, Vaadin starts the Vite development server on the next application run, enabling frontend hot-deployment.
 
 
 [[flow-to-hilla]]
```
Comment (tltv) [inline articles/flow/integrations/hilla.adoc:67]: flow to hilla instruction is not working when using skeleton-starter-hilla-react example project (branch v24.4, snapshot or latest alpha). Following steps to add Flow view would end up getting exception that instructs to update routes.tsx properly. This is what it works with:
```
import MainView from "Frontend/views/MainView.js";
import {
    createBrowserRouter,
    RouteObject
} from "react-router-dom";
import { buildRoute } from "Frontend/generated/flow/Flow";

let routing = [
  { path: "/", element: <MainView /> },
];
export const routes = buildRoute(routing);
export const router = createBrowserRouter(routes, {basename: new URL(document.baseURI).pathname });
```

Either link to the starter should be different, or buildRoute() should be mentioned here, or starter should be changed.

**Review by tltv — CHANGES_REQUESTED** [review]

File: articles/flow/integrations/hilla.adoc (hunk 2 above)
Comment (mshabarov) [reply articles/flow/integrations/hilla.adoc:67]: Works with these changes https://github.com/vaadin/skeleton-starter-hilla-react/pull/182

Comment (mshabarov) [reply articles/flow/integrations/hilla.adoc:67]: You need to rename to `$index.tsx` to make main view to work with FS router.

**Review by tltv — APPROVED** [review]

File: articles/flow/integrations/hilla.adoc (hunk 2 above)
Comment (tltv) [reply articles/flow/integrations/hilla.adoc:67]: Works fine with the updated branch.

**Pull request merged**